spec/rubocop/cop/rspec/stubbed_mock_spec.rb

```diff
@@ -126,12 +126,49 @@
     RUBY
   end
 
-  it 'tolerates passed arguments without parentheses' do
+  it 'flags even when passed arguments without parentheses' do
     expect_offense(<<~RUBY)
       expect(Foo)
       ^^^^^^^^^^^ Prefer `allow` over `expect` when configuring a response.
         .to receive(:new)
         .with(bar).and_return baz
     RUBY
   end
+
+  it 'flags `should_receive`', skip: 'Not implemented yet' do
+    expect_offense(<<~RUBY)
+      foo.should_receive(:bar).and_return(:baz)
+      ^^^^^^^^^^^^^^^^^^^ Prefer `allow` over `should_receive` when configuring a response.
+    RUBY
+  end
+
+  it 'flags `should_not_receive`',
+     skip: 'Not sure what to do with this kind of expectation' do
+    expect_offense(<<~RUBY)
+      foo.should_not_receive(:bar)
+      ^^^^^^^^^^^^^^^^^^^^^^ Prefer `allow(...).not_to receive` over `should_not_receive` when configuring a response.
+    RUBY
+  end
+
+  it 'flags `are_expected`',
+     skip: 'This passed before this commit. Handled like is_expected?' do
+    expect_offense(<<~RUBY)
+      are_expected.to receive(:bar).and_return(:baz)
+      ^^^^^^^^^^^^ Prefer `` over `are_expected` when configuring a response.
+    RUBY
+  end
+
+  it 'flags `should`', skip: 'Not implemented yet' do
+    expect_offense(<<~RUBY)
+      foo.should receive(:bar).and_return(:baz)
```

Comment:
RSpec 2 legacy, deprecated a decade ago and removed in next major RSpec. We don’t want to support this.